### ISSUE TITLE
Cannot get new _layout directory to play nice with nunjuck extends

### ISSUE THREAD

**Issue by jeromecoupe** [11ty/eleventy#478]
I must do something stupid but I cannot get the separate root `_layout` folder to work.
Had everything working fine using a `layouts` folder inside the `_include` directory.

Using Eleventy version 0.8.1 
Here is my current project structure

```
src
+-- _data
+-- _includes
+-- _layouts
    +-- base.njk
    +-- blogpost.njk
+-- blogposts
    +-- yyyy-mm-dd-title-blogpost.md
    +-- other blogposts ...
    +-- blogposts.json (layout specified here)
+-- pages
    +-- index.njk
    +-- other pages ...
```

- Changed my config as [per the doc](https://www.11ty.io/docs/config/#directory-for-layouts-(optional)) 
- Moved `layouts` folder out of of `_includes` and renamed it `_layouts`
- My pages previously worked with `{% extends "layouts/base.njk" %}`

I expected that it would now work with `{% extends "base.njk" %}` since Eleventy would now look in the `_layouts` folder instead of in `_includes` but I had the following error:

```
Having trouble rendering njk template ./src/pages/index.njk (TemplateContentRenderError)
(unknown path)
`Error: template not found`.
```

I tried a whole bunch of paths: `{% extends "_layouts/base.njk" %}` as well as relative and absolute ones, not to avail. Looking at the `_includes` directory seems consistent with the [Nunjucks supported features page](https://www.11ty.io/docs/languages/nunjucks/#supported-features). IMHO it would be logical for Nunjucks to look in the directory specified by the `dir.layouts` key for extends if you specify one.

That being said, maybe there is something I don't understand in the way Eleventy and Nunjucks interact here. Would appreciate your insight.

**Comment by jeromecoupe** [comment]
Ok then. That's a bit of a shame but I can live with it ;o). If not too much work, would be something nice to consider in the future.

**Comment by jevets** [comment]
@zachleat Could we use an array of paths when instantiating Nunjucks (if `config.dir.layouts` is defined)?

```
nunjucks.configure([
  // 11ty's config.dir.includes,
  // 11ty's config.dir.layouts,
])
```

I haven't found a way to access the configured Nunjucks instance/env from Eleventy's config file. (I've concluded that it hasn't even been instantiated yet when .eleventy.js is loaded). It could be useful to access the configured nunjucks environment later on (especially w/ regards to #189).

**Comment by zachleat** [comment]
Hey @jevets if you’d like to discuss that further can you open a new issue?

Going to close this as @jevets fixed this in the docs at #516. See https://www.11ty.io/docs/config/#directory-for-layouts-(optional)
